project_gtd/wizard/project_gtd_empty.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2004-2010 Tiny SPRL <http://tiny.be>.
+# Copyright 2017 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import exceptions, fields, models
+from odoo.tools.translate import _
+
+
+class ProjectTimeboxEmpty(models.TransientModel):
+    _name = "project.timebox.empty"
+    _description = "Project Timebox Empty"
+
+    state = fields.Selection(
+        [("init", "init"), ("done", "done")],
+        readonly=True,
+        default="init",
+    )
+
+    def process(self):
+        close = []
```

Comment:
Could you please documenting a bit  the method by using a docstring? Not totally clear what `up` is used for?

Reply:
this is just a migration of the module, I did not go into the details of the implementation.